.github/ISSUE_TEPLATE/bug_report.yml

```diff
@@ -0,0 +1,56 @@
+name: Bug Report
+description: Create a report to help us improve
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for filing a 🐛 bug report 😄!
+  - type: textarea
+    id: problem
+    attributes:
+      label: Problem
+      description: >
+        Please provide a clear and concise description of what the bug is,
+        including what currently happens and what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps
+      description: Please list the steps to reproduce the bug.
+      placeholder: |
+        1.
+        2.
+        3.
+  - type: textarea
+    id: scarb-version
+    attributes:
+      label: Scarb Version
+      description: >
+        Please paste the output of running `scarb --version`.
+        Remember to do this in the affected project directory.
+      render: text
+    validations:
+      required: true
+  - type: textarea
+    id: cargo-version
+    attributes:
+      label: Cargo Version
```

Comment:
Why do we need cargo version if we're going to ship binaries?

Reply:
I just thought that it would be better to create a template regarding current shipping process, but you're right. It's probably one of the first things we are going to change.